analyzers/tests/SonarAnalyzer.UnitTest/Rules/ArrayPassedAsParamsTest.cs

```diff
@@ -44,9 +44,13 @@ public class ArrayPassedAsParamsTest
     [TestMethod]
     public void ArrayPassedAsParams_CSharp12() =>
         builderCS.AddSnippet("""
-            MyClass s = new(1, [1, 2, 3]); // FN
+            MyClass s0 = new(1, [1, 2, 3]);     // Noncompliant
+            MyClass s1 = new(1, []);            // Noncompliant
+            MyClass2 s2 = new([1], [1, 2, 3]); // Noncompliant
```

Comment:
It would be good to vertically align the third `// Noncompliant` comment to the first two.